showcase/src/assets/examples/toast/toast.html

```diff
@@ -0,0 +1,7 @@
+<button class="toast-demo-button" color="primary" mat-raised-button (click)="openToast(exampleMessage)">{{ 'SHOWCASE.DEMO.TOAST.BUTTON_INFO' | translate }}</button>
```

Comment:
On the example files we should put the labels in English instead of the translation keys ;)